lib/redmine.js

```diff
@@ -99,7 +105,7 @@ Redmine.prototype.request = function(method, path, params, callback) {
 
   // impersonate to a login user
   if (this.impersonate) {
```

Comment:
How can this work ? This should be `if (this.config.impersonate)` since there is no code that sets `this.impersonate` right ?